### PR TITLE
Updated Renci.SshNet library and fixed tests

### PR DESCRIPTION
**Pull request by RikuVirtanen**
Updated Renci.SshNet library and fixed tests by deleting some obsolete test files.

### CODE REVIEW

File: Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/FileTransporter.cs [hunk 1]
```diff
@@ -88,6 +88,9 @@ public FileTransferResult Run(CancellationToken cancellationToken)
 
                     using (var client = new SftpClient(connectionInfo))
                     {
+                        client.ConnectionInfo.KeyExchangeAlgorithms.Remove("curve25519-sha256");
```
Comment (jefim) [inline Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/FileTransporter.cs:91]: Can you add an explanation in a comment please?

Comment (RikuVirtanen) [reply Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/FileTransporter.cs:92]: fixed

File: .github/workflows/UploadFiles_build_and_test_on_main.yml [hunk 2]
```diff
@@ -14,6 +14,6 @@ jobs:
     uses: FrendsPlatform/FrendsTasks/.github/workflows/linux_build_main.yml@main
     with:
       workdir: Frends.SFTP.UploadFiles
-      prebuild_command: docker-compose -f ./Frends.SFTP.UploadFiles.Tests/docker_compose.yml up -d
+      prebuild_command: docker-compose -f ./Frends.SFTP.DownloadFiles.Tests/docker_compose.yml up -d
```
Comment (Svenskapojkarna) [inline .github/workflows/UploadFiles_build_and_test_on_main.yml:17]: I think this is a mistake.

Comment (RikuVirtanen) [reply .github/workflows/UploadFiles_build_and_test_on_main.yml:17]: Fixed

**Pull request merged**